presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -667,6 +667,9 @@ public void commitCreateTable(ConnectorSession session, ConnectorOutputTableHand
             rollbackPartitionUpdates(partitionUpdates, "table creation");
             throw throwable;
         }
+        finally {
+            partitionCommitter.close();
```

Comment:
This is also a false positive. The `close()` method on `PartitionCommitter` should be named `flush()` (and the object should not be `Closeable`). It would not make sense to call it after `abort()`.

Will update `PartitionCommitter`. Thanks!